### ISSUE TITLE
Multiple morphological abnormalities of the flagella

### ISSUE THREAD

**Issue by MagoWyr** [obophenotype/human-phenotype-ontology#9829]
**Preferred term label:**
Multiple morphological abnormalities of the flagella (MMAF)

**Synonyms**

**Definition (free text, please give PubMed ID)**
The recently coined phenotype “Multiple morphological abnormalities of the flagella” (“MMAF”) describes a combination of abnormal sperm motility and abnormal sperm morphology. It should only be used if the sperm are uniformly and specifically altered and show consistent immobility or decreased motility. 

**Parent term (use hpo.jax.org/app)**
HP:0012207, HP:0012868

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA: 276234, ORPHA: 399813, ORPHA: 399808

**Your nano-attribution (ORCID)**
0000-0002-7651-2403

**Comment by pnrobinson** [comment]
@MagoWyr 
this is a so-called "bundled" observation that is closer to a disease description than an atomic phenotypic abnormality, which is the scope of the HPO. The way of coding this using HPO would be to code each of the anomalies separately.
